src/model/FeatureCol.js

```diff
@@ -68,7 +68,7 @@ const FeatureCol = Collection.extend({
 
   getCurrentHeight: function() {
     if (this.length === 0) {
-      return 1;
+      return 0;
```

Comment:
Remove the extra space at the bottom of the column ruler if no annotations are present.